public/js/components/Scopes.js

```diff
@@ -39,7 +39,7 @@ function getSpecialVariables(pauseInfo, parentName) {
     vars.push({
       name: "<exception>",
       path: parentName + "/<exception>",
-      contents: { value: thrown.toJS() }
+      contents: { value: thrown }
```

Comment:
Oh, I just realized something: you're tests are throwing strings. Make them throw `Error` objects instead and you'll see what happens. But that is a good edge case: strings aren't converted to immutable... so we need to only convert to JS if needed.